compiler/rustc_target/src/spec/targets/aarch64_be_unknown_linux_musl.rs

```diff
@@ -0,0 +1,37 @@
+use rustc_abi::Endian;
+
+use crate::spec::{SanitizerSet, StackProbeType, Target, TargetMetadata, TargetOptions, base};
+
+pub(crate) fn target() -> Target {
+    let mut base = base::linux_musl::opts();
+    base.max_atomic_width = Some(128);
+    base.supports_xray = true;
+    base.features = "+v8a".into();
+    base.stack_probes = StackProbeType::Inline;
+    base.supported_sanitizers = SanitizerSet::ADDRESS
+        | SanitizerSet::CFI
+        | SanitizerSet::LEAK
+        | SanitizerSet::MEMORY
+        | SanitizerSet::THREAD;
+
+    // FIXME(compiler-team#422): musl targets should be dynamically linked by default.
+    base.crt_static_default = true;
+
+    Target {
+        llvm_target: "aarch64_be-unknown-linux-musl".into(),
+        metadata: TargetMetadata {
+            description: Some("ARM64 Linux with musl 1.2.3 (big-endian)".into()),
+            tier: Some(3),
+            host_tools: Some(true),
```

Comment:
This should be false because host tools aren't built for this target (possible, but it's not in tier 2)

Reply:
done